packages/components/src/dropdown/index.js

```diff
@@ -40,6 +42,15 @@ class Dropdown extends Component {
 		this.container = ref;
 	}
 
+	/**
+	 * When contents change height due to user interaction,
+	 * `refresh` can be called to re-render Popover with correct
+	 * attributes which allow scroll, if need be.
+	 */
+	refresh() {
+		this.popoverRef.current.refresh();
```

Comment:
Should we check for the existence of the popover before calling refresh here. It seems that the popover is not always rendered :)